aztec_backend_wasm/src/lib.rs

```diff
@@ -21,7 +23,7 @@ pub fn compute_witnesses(
 ) -> ComputedWitness {
     console_error_panic_hook::set_once();
 
-    let circuit: Circuit = circuit.into_serde().unwrap();
+    let circuit: Circuit = JsValueSerdeExt::into_serde(&circuit).unwrap();
```

Comment:
Why the change from method to function syntax? Is there another `into_serde` on the circuit type?

Reply:
This is matching a change we performed in https://github.com/noir-lang/noir/pull/605

tl;dr: there's a circular dependency in `wasm-bindgen` which is resolved by doing this.